src/Middleware/HttpLogging/src/HttpLoggingFields.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.HttpLogging
+{
+    /// <summary>
+    /// Logging enum to enable different request and response logging fields.
+    /// </summary>
+    [Flags]
+    public enum HttpLoggingFields
```

Comment:
How big could this enum get? Should its type be a long?

Reply:
May be a decent idea for future proofing.